composer.json

```diff
@@ -27,16 +27,16 @@
 		}
 	],
 	"require": {
-		"php": ">=5.6.0",
+		"php": ">=7.2.29",
 		"klaussilveira/gitter": "dev-php-semver-checker",
 		"symfony/console": "^2.7|^3.0|^4.0",
 		"tomzx/finder": "^0.1",
-		"tomzx/php-semver-checker": "^0.12",
+		"tomzx/php-semver-checker": "^0.14",
 		"vierbergenlars/php-semver": "^3.0"
 	},
 	"require-dev": {
 		"mockery/mockery": "^1.0",
-		"phpunit/phpunit": "^4.0|^5.0"
+		"phpunit/phpunit": "^7.0|^8.0|^9.0"
```

Comment:
**SHOULD**
PHPUnit 8 and 9 are currently supported, we should drop ^7.0.

Reply:
Good deal - I think that this may have been to match tomzx/gitter changes.
So I can rework that PR for PHP >=7.2 as well and drop PHPunit 7.0 from both.